package.json

```diff
@@ -51,7 +51,7 @@
     "@map-colonies/error-types": "^1.1.0",
     "@map-colonies/express-access-log-middleware": "^0.0.3",
     "@map-colonies/js-logger": "^0.0.3",
-    "@map-colonies/mc-model-types": "^12.1.0",
+    "@map-colonies/mc-model-types": "file:../mc-models/map-colonies-mc-model-types-13.1.3.tgz",
```

Comment:
replcae when mc-models will be published

Reply:
fix lint and tests

Reply:
remove auto-generated ID creation

Reply:
id default value removed from db schema